README.md

```diff
@@ -135,6 +135,19 @@ python generate_custom_image.py \
     customization script. See more information about VM metadata on
     https://cloud.google.com/sdk/gcloud/reference/compute/instances/create.
 
+#### Override properties
+
+Custom images can also be used to overwrite properties set during cluster creation.
+Users can create file `dataproc.custom.properties` in `/etc/google-dataproc` using customization
+script. Sample contents of file
```

Comment:
Perhaps replace example with this customization script bash snippet: 

Sample customization script file-creation snippet to override two
     cluster properties:
<pre>
     cat &lt;&lt;EOF &gt;/etc/google-dataproc/dataproc.custom.properties
     dataproc.conscrypt.provider.enable=true
     dataproc.logging.stackdriver.enable=false
     EOF
</pre>

Reply:
+1

Reply:
Fixed.